compact/ecal_barrel_sciglass_tiltless.xml

```diff
@@ -0,0 +1,100 @@
+<lccdd>
+  <comment>
+    Implementation of the barrel SciGlass calorimeter
+  </comment>
+
+  <define>
+    <documentation>
+      The EcalBarrel eta limits cut into the barrel shape.
+    </documentation>
+    <constant name="EcalBarrel_etamin" value="-3.5"/>
+    <constant name="EcalBarrel_etamax" value="+1.4"/>
+  </define>
+
+  <display>
+    <vis name="det_vis" alpha="1.0" r="1.0" g="0.0" b="0.0" showDaughters="true" visible="false" />
+    <vis name="becal_wedge" alpha="1.1" r="0.82" g="0.84" b="0.86" showDaughters="true" visible="true" />
+    <vis name="carbon_fiber" alpha="0.9" r="0.1" g="0.1" b="0.1" showDaughters="true" visible="true" />
+    <vis name="family1" alpha="1.0" r="0.22" g="0.54" b="0.71" showDaughters="true" visible="true" />
+    <vis name="family2" alpha="1.0" r="0.90" g="0.48" b="0.47" showDaughters="true" visible="true" />
+    <vis name="family3" alpha="1.0" r="0.87" g="0.49" b="0.09" showDaughters="true" visible="true" />
+    <vis name="family4" alpha="1.0" r="0.94" g="0.48" b="0.73" showDaughters="true" visible="true" />
+    <vis name="family5" alpha="1.0" r="0.44" g="0.63" b="0.44" showDaughters="true" visible="true" />
+    <vis name="family6" alpha="1.0" r="0.50" g="0.35" b="0.76" showDaughters="true" visible="true" />
+  </display>
+
+  <detectors>
+    <comment>
+      An EM calorimeter with SciGlass blocks
+    </comment>
+
+    <detector id="4" name="EcalBarrel" type="epic_SciGlassCalorimeterTiltless" readout="EcalBarrelHits" vis="det_vis" calorimeterType="EM_BARREL">
+      <comment>
+        Global detector dimensions from https://eic.jlab.org/Geometry/Detector/Detector-20220624102507.html
+      </comment>
+
+      <dimensions rmin="EcalBarrel_rmin" rmax="EcalBarrel_rmax"
+                  zmax="195 * cm" zmin="-285 * cm"
+                  etamin="EcalBarrel_etamin" etamax="EcalBarrel_etamax"/>
+
+      <wedge-box inner_r="80.0 * cm" thickness="2 * mm" vis="becal_wedge" material="Aluminum5083" />
```

Comment:
While a `-` as element name is allowed, it is uncommon and some xml processing software may not like it (in particular homegrown stuff). Can we just use `wedge_box` instead (here and in the plugin)?

Reply:
Same with `carbon-fiber-support` and `cut-out`.